src/Oci8/Schema/Grammars/OracleGrammar.php

```diff
@@ -158,7 +158,7 @@ protected function addPrimaryKeys(Blueprint $blueprint)
      */
     public function compileTableExists()
     {
-        return "select * from user_tables where upper(table_name) = upper(?)";
+        return "select * from all_tables where upper(table_name) = upper(?)";
```

Comment:
We need to override `hasTable` method on `OracleBuilder` to be able to insert the schema binding and migration works.

```php
// Schema/Grammars/OracleGrammar
    public function compileTableExists()
    {
        return "select * from all_tables where upper(owner) = upper(?) and upper(table_name) = upper(?)";
    }

// Schema/OracleBuilder.php
    /**
     * Determine if the given table exists.
     *
     * @param  string $table
     * @return bool
     */
    public function hasTable($table)
    {
        $sql = $this->grammar->compileTableExists();

        $database = $this->connection->getConfig('username');
        $table    = $this->connection->getTablePrefix() . $table;

        return count($this->connection->select($sql, [$database, $table])) > 0;
    }
```